src/foam/nanos/dig/DigUtil.java

```diff
@@ -0,0 +1,78 @@
+package foam.nanos.dig;
```

Comment:
License header please.

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/2832/commits/d03029976dee996b104fa23834a95c877202baed.